### PR TITLE
Increase default number of targeted sweep shards

### PR DESCRIPTION
**Pull request by gmaretic**
**Goals (and why)**:
This enables us to increase parallelism if necessary to process existing entries faster, as opposed to only being able to do so for future writes.

### CODE REVIEW

**Comment by gmaretic** [comment]
@tpetracca for SA: in the future we will be able to speed up targeted sweep

File: atlasdb-config/src/main/java/com/palantir/atlasdb/config/TargetedSweepRuntimeConfig.java [hunk 1]
```diff
@@ -42,7 +42,7 @@ public boolean enabled() {
      */
     @Value.Default
     public int shards() {
-        return AtlasDbConstants.DEFAULT_SWEEP_QUEUE_SHARDS;
```
Comment (tpetracca) [inline atlasdb-config/src/main/java/com/palantir/atlasdb/config/TargetedSweepRuntimeConfig.java]: does this constant have any other references? should we delete it from that class?

Comment (gmaretic) [reply atlasdb-config/src/main/java/com/palantir/atlasdb/config/TargetedSweepRuntimeConfig.java]: It is used in tests and, more importantly, as the default value when reading the persisted number of shards and no value is present -- we take the min of this and the runtime config.

**Comment by gmaretic** [comment]
Intentionally did not increase the number of threads, so we have wiggle room to increase speed when necessary. Otherwise, by the next time we realise we are falling behind it will again be too late. (Note that by increasing the number of shards, we are actually increasing parallelism already as we have one thread per host doing the sweeping)

File: atlasdb-config/src/main/java/com/palantir/atlasdb/config/TargetedSweepRuntimeConfig.java (hunk 1 above)
Comment (jeremyk-91) [reply atlasdb-config/src/main/java/com/palantir/atlasdb/config/TargetedSweepRuntimeConfig.java]: maybe leave a comment as to why it's not the default? Logic makes sense, but it's still a bit surprising

**Pull request merged**